website/docs/d/pi_datacenter.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_datacenter"
+description: |-
+  Manages a Datacenter in the Power Virtual Server cloud.
+---
+
+# ibm_pi_Datacenter
+
+Retrieve information about a Power Systems Datacenter.
+
+## Example usage
+
+```terraform
+data "ibm_pi_datacenter" "datacenter" {
+  pi_datacenter_zone= "dal12"
+}
+```
+  
+## Argument reference
+
+Review the argument references that you can specify for your data source.
+
+- `pi_datacenter_zone` - (Optional, String) Datacenter zone you want to retrieve. If no value is supplied, the `zone` configured within the IBM provider will be utilized.
+
+## Attribute reference
+
+In addition to all argument reference list, you can access the following attribute references after your data source is created.
+
+- `pi_datacenter_capabilities` - (Map) Datacenter Capabilities.
+
+    Nested schema for `pi_datacenter_capabilities`:
+  - `cloud-connections` - (Bool) Cloud-connections capability.
+  - `disaster-recovery-site` - (Bool) Disaster recovery site.
+  - `power-edge-router` - (Bool) Power edge router capability.
+  - `vpn-connections`- (Bool) VPN-connections capability.
+  - `sysdig-enabled`- (Bool) Sysdig-enabled capability.
+
+- `pi_datacenter_location` - (Map) Datacenter location.
+
+    Nested schema for `pi_datacenter_location`:
+  - `region` - (String) The Datacenter location region zone.
+  - `type` - (String) The Datacenter location region type.
+  - `url`- (String) The Datacenter location region url.
+- `pi_datacenter_status` - (String) The Datacenter status, `ACTIVE`,`MAINTENANCE` or `DOWN`.
+- `pi_datacenter_type` - (String) The Datacenter type, `Public Cloud` or `Private Cloud`.
+
+## Notes
```

Comment:
Can we move this `notes` in `Example usage` section? eg: https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/d/pi_image.html.markdown

Reply:
Moved `notes` in all of them